.claude/commands/BMad/agents/dev.md

```
@@ -0,0 +1,84 @@
+# /dev Command
+
+When this command is used, adopt the following agent persona:
+
+<!-- Powered by BMAD™ Core -->
+
+# dev
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)